### PR TITLE
Feature/profile redesign menu edits

### PR DESCRIPTION
**Pull request by clubkemp**
hiding future features in menu
fixing create post button z-index
adding eslint ignore to postTab

### CODE REVIEW

File: client/src/components/Feed/PostTabCard.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ const PostTabCard = ({
   useEffect(() => {
     const initialTab = tabs.find((tab) => tab.title === initialPage);
     setCurrentTab(initialTab);
-  }, [cardContents, initialPage, tabs]);
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
```
Comment (linadinh2602) [inline client/src/components/Feed/PostTabCard.js]: removing the dependency would cause bugs. What is the problems are you trying to fix?

Comment (clubkemp) [reply client/src/components/Feed/PostTabCard.js]: updated dependency list to `[cardContents, initialPage]`

**Review by aditilamkhede — CHANGES_REQUESTED** [review]
@clubkemp looks Ok, just a quick update, Can we make fon-size of Posts Navigation little bigger.

File: client/src/components/Feed/PostTabCard.js [hunk 2]
```diff
@@ -162,7 +162,7 @@ const PostTabCard = ({
   useEffect(() => {
     const initialTab = tabs.find((tab) => tab.title === initialPage);
     setCurrentTab(initialTab);
-  }, [cardContents, initialPage, tabs]);
+  }, [cardContents, initialPage]); // eslint-disable-line react-hooks/exhaustive-deps
```
Comment (linadinh2602) [inline client/src/components/Feed/PostTabCard.js:165]: I already covered this in my PR

**Pull request merged**